src/components/Header/Modal/NotificationModal/index.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import CloseRoundedIcon from '@mui/icons-material/CloseRounded';
+import { Button } from '@/components/Button';
+import { Avatar } from '@/components/Avatar';
+import { BackgroundBox } from '@/components/BackgroundBox';
+import { Text } from '@/components/Text';
+import { Seperator } from '@/components/Seperator';
+
+const dummyData = [
+  {
+    nickname: '고양이가 멍멍',
+    image: 'https://source.unsplash.com/random',
+    postId: 100,
+    status: '목격',
+    checked: true
+  },
+  {
+    nickname: '성격 드러운 고먐미',
+    image: 'https://source.unsplash.com/daily',
+    postId: 101,
+    status: '완료',
+    checked: false
+  },
+  {
+    nickname: '어벤져스에서아이언멘은',
+    image: 'https://source.unsplash.com/weekly',
+    postId: 102,
+    status: '발견',
+    checked: false
+  },
+  {
+    nickname: '블랑킷을입은양탄자를',
+    image: 'https://source.unsplash.com/category/nature',
+    postId: 103,
+    status: '실종',
+    checked: true
+  }
+];
+
+const NotificationModal = ({ visible }) => {
+  return (
+    <Wrapper visible={visible}>
+      <BackgroundBox width="34rem">
+        <TopContainer>
+          <TextWrapper>전체삭제</TextWrapper>
+        </TopContainer>
+        <MiddleContainer>
+          <NotificationList aria-labelledby="notificationButton">
+            {dummyData.map(({ nickname, image, postId, status, checked }, index) => {
+              return (
+                <>
+                  <Seperator type="horizon" key={index} />
```

Comment:
```suggestion
                <li key={index}>
                  <Seperator type="horizon" />
```
wrapper에 key가 들어가야 합니다